migrations/env.py

```diff
@@ -90,8 +90,14 @@ def run_migrations_offline():
     script output.
 
     """
+    def include_object(object, name, type_, reflected, compare_to):
+        return not (type_ == "index" or type_== "table" and reflected and compare_to is None)
+
     url = config.get_main_option("sqlalchemy.url")
-    context.configure(url=url)
+    context.configure(
+        url=url,
+        include_object=include_object,
+        target_metadata=target_metadata,)
```

Comment:
What was the point of adding these extra things to the context? What were we missing and what do we gain by adding these?